### ISSUE TITLE
Installed command fails to work when arguments contain special symbols

### ISSUE THREAD

**Issue by yamaton** [yamaton/condax#16]
## Description

After installation with `condax`, a command fails to work when the arguments contain special symbols.

## Reproduce

```shell
$ condax install bat
$ echo foo > "(.txt"
$ bat "(.txt"
/tmp/mambafiQc6ufyQVG: line 5: syntax error near unexpected token `('
/tmp/mambafiQc6ufyQVG: line 5: `exec /home/vierbein/.local/share/condax/envs/bat/bin/bat (.txt'
```

## Context

I suppose this problem is because `"$@"` in `~/.local/bin/bat` (or other scripts there) does not quote arguments properly.

**Comment by yamaton** [comment]
It may be an issue in the upstream: https://github.com/mamba-org/mamba/issues/1936

**Comment by yamaton** [comment]
Fixed in `mamba` 0.26.0.
